extensions/amp-dailymotion/0.1/amp-dailymotion.js

```diff
@@ -70,6 +159,66 @@ class AmpDailymotion extends AMP.BaseElement {
   }
 
   /** @private */
+  handleEvents_(event) {
+    if (event.origin != 'https://www.dailymotion.com' ||
+        event.source != this.iframe_.contentWindow) {
+      return;
+    }
+    if (!event.data || !event.type || event.type != 'message') {
+      return;  // Event empty
+    }
+    const data = parseQueryString(event.data);
+    if (data === undefined) {
+      return; // The message isn't valid
+    }
+    if (data.event == PlayerEvents.API_READY) {
+      this.playerReadyResolver_(true);
+    }
+
+    console.log(data);
+
+    switch (data.event) {
+      case PlayerEvents.END:
+      case PlayerEvents.PAUSE:
+        this.element.dispatchCustomEvent(VideoEvents.PAUSE);
+        this.playerState_ = PlayerEvents.PAUSE;
+        break;
+      case PlayerEvents.PLAY:
+        this.element.dispatchCustomEvent(VideoEvents.PLAY);
+        this.playerState_ = PlayerEvents.PLAY;
+        break;
+      case PlayerEvents.VOLUMECHANGE:
+        const muteEvent = (
+          (data.volume == 0 || data.muted == 'true') ?
+          VideoEvents.MUTED : VideoEvents.UNMUTED
+        );
+        this.element.dispatchCustomEvent(muteEvent);
```

Comment:
we need to make sure we only dispatch this when muted state changed. I think this will dispatch multiple `UNMUTED` events when user changes volume for example. amp-youtube keeps current muted state and compares before dispatching, we can do the same here.